src/snowflake/cli/plugins/git/manager.py

```diff
@@ -0,0 +1,8 @@
+from snowflake.cli.api.sql_execution import SqlExecutionMixin
+
+
+class GitManager(SqlExecutionMixin):
+    def show(self, object_type: str, repo_name: str):
+        assert object_type in ["branches", "tags"]
```

Comment:
Personally I would separate this into two methods. If not, then let's introduce enum for branches and tags

Reply:
I've separated the commands - An enum would be an overkill for now.